packages/common/mock/pubnub-mock.ts

```diff
@@ -8,16 +8,20 @@ import type {
   SignalResponse,
   SendFileResponse,
 } from "pubnub";
-import users from "../../data/users/users.json";
-import messages from "../../data/messages/lorem.json";
-import channels from "../../data/channels/work.json";
+import users from "../../../data/users/users.json";
```

Comment:
Have we ever considered a module resolver? So that instead of thinking how many "backs" we have to make we could just set some directories as entrypoints so it'd be e.g. import users from @data/users/users.json.

P.S. I know that the mock folder will probably also be removed but just wondering in general.

Reply:
I surfaced the data package in the repo as `@pubnub/sample-chat-data` via yarn workspaces fairly recently. It can probably be used here, so just an oversight. I only remember sample apps having a problem with this approach when they were installed independently (old npm version installing deps directly in the app folder)